com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/pyfunction.pm

```diff
@@ -26,6 +26,8 @@ sub splpy_tuplestyle{
     $pystyle = 'pickle';
  } elsif ($attr->getName() eq 'string') {
     $pystyle = 'string';
+ } elsif ($attr->getName() eq 'message') {
```

Comment:
I don't think we want a style specific to a toolkit's operator. THis can be changed to use the SPL dict approach being added in pr #400